bfdd/bfd.c

```diff
@@ -2065,3 +2071,14 @@ unsigned long bfd_get_session_count(void)
 {
 	return bfd_key_hash->count;
 }
+
+void bfd_rtt_init(struct bfd_session *bfd)
+{
+	uint8_t i;
+
+	/* initialize RTT */
+	bfd->rtt_valid = 0;
+	bfd->rtt_index = 0;
+	for (i = 0; i < BFD_RTT_SAMPLE; i++)
+		bfd->rtt[i] = 0;
```

Comment:
replace < BFD_RTT_SAMPLE with
< sizeof(bfd->rtt).
It is more generic

Reply:
done